treeherder/etl/taskcluster_pulse/handler.py

```diff
@@ -111,6 +111,23 @@ def validateTask(task):
     taskId = message["payload"]["status"]["taskId"]
     asyncQueue = taskcluster.aio.Queue({"rootUrl": message["root_url"]}, session=session)
     task = (await asyncQueue.task(taskId)) if not taskDefinition else taskDefinition
+
+    # Bug 1590512 - A more general solution is needed to avoid using env variables that
```

Comment:
Eventually this code needs to turn into the code that excludes all tasks that we don't care about.

The tasks being excluded are actually valid for Treeherder ingestion. We just don't want them even though they have the right routes.

I will be cleaning up this code in [bug 1560596](https://bugzilla.mozilla.org/show_bug.cgi?id=1560596)

I'm trying hard not to make any more changes to this PR because it is currently expensive to have to revalidate via `prototype`.